onnxruntime/core/session/ort_env.cc

```diff
@@ -19,6 +19,10 @@
 using namespace onnxruntime;
 using namespace onnxruntime::logging;
 
+namespace onnxruntime {
+void UnloadSharedProviders();
+}
+
```

Comment:
This should be a header. We had an issue with local prootypes in pybind. Took some time to figure out linking error.